README.md

```diff
@@ -167,6 +167,10 @@ If you make refund from the Midtrans Dashboard, Refund notification is sent to M
     
     The status change may not mean that the refund has carried out successfully on Midtrans side. When the transaction status in Midtrans dashboard changes to REFUND, then the refund went through successfully
 </details>
+
+### Add Payment Fee
+Payment fees can be enable by using external extension. You can use [Mageprince Magento2 PaymentFee](https://github.com/mageprince/magento2-paymentfee/) extension. Payment Fee extension for Magento 2 allows adding extra charges for specific payment methods and displays them on the cart page, checkout page, invoice, and credit memo.
+All fee calculation will handle by external extension. We only put the PaymentFee value from Magento order object as items. 
     
```

Comment:
Likely we will also get asked on tutorial/example on how to use that plugin to add fee, so it's better to show write the steps/screenshots. And put it into a "click to expand" section (like [this](https://github.com/veritrans/SNAP-Woocommerce#customizing-snap-api-parameters))

Reply:
Update, please help to review. Regarding the steps of installation and configuration that plugins. Mostly I put their GitHub readme. So if any update from their side, we don't need to update our readme